docs/HTML.md

```diff
@@ -0,0 +1,203 @@
+### 1. `<hr>` -- horizontal rule (dividing line)
+
+* *type:* block-level
+* *content:* none, empty element
+* *support:* all browsers
+* *example:*
+```
+<p>some text</p>
+<hr>
+<p>text down under</p>
+```
+<p>some text</p>
+<hr>
+<p>text down under</p>
+
+### 2. `<table>` -- HTML Table element represents data in 2 dimensions or more
+
+* *type:* block-level
+* *content:* Flow Content[1]
+* *support:* all browsers
+* *example:*
+```
+<table>
+<tr>
+<td>top left</td>
+<td>top right</td>
+</tr>
+<tr>
+<td>bottom left</td>
+<td>bottom right</td>
+</tr>
+</table>
+```
+<table>
+<tr>
+<td>top left</td>
+<td>top right</td>
+</tr>
+<tr>
+<td>bottom left</td>
+<td>bottom right</td>
+</tr>
+</table>
```

Comment:
Well. You sure bit off a doozie. To be thorough, you might consider adding the other related tags... Except there's almost 10 by themselves. We'll cover this family elsewhere, but since you brought it up...